src/utils/style-formatter.js

```diff
@@ -23,27 +23,26 @@ const backgroundPosition = {
 
 const formatProperty = (path, setting) => `${path}: ${setting};`;
 
-const getColor = ({ useColorExpression, colorExpression, color }, defaultColor, palette) => {
+const getColor = ({ useColorExpression, colorExpression, color }, defaultColor, theme) => {
   const resolvedColor = useColorExpression
     ? colorUtils.resolveExpression(colorExpression)
-    : colorUtils.resolveColor(color, palette);
-  return resolvedColor === 'none' ? defaultColor : resolvedColor;
+    : theme.getColorPickerColor(color);
+  return !resolvedColor || resolvedColor === 'none' ? defaultColor : resolvedColor;
```

Comment:
Is this where you were having trouble with none?

Reply:
kind of - the default palette used in Sense has `"none"` as the first 'color' which causes all color indexes to be shifted by one - that's fine though since we should be able to handle palettes of different sizes anyway